src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -244,4 +254,34 @@ public boolean runUntilEnd(Path appPath, String appName) throws IOException, Int
         return true;
     }
 
+    // Default settings below, can be overridden by subclasses
+
+    String getAdditionalSourceFileLocation() {
+        return "/native/linux/";
+    }
+
+    List<String> getAdditionalSourceFiles() {
+        return Arrays.asList("launcher.c","thread.c");
```

Comment:
This seems to be a constant. Why are we creating this list all the time by calling a method?

Reply:
It's not constant. It depends on the Configuration. However, for each configuration it is a constant.

Reply:
Ah, you probably meant to change the Arrays.asList() into a constant for this class. Fixed.